source/views/layouts/govuk_template.html.erb

```diff
@@ -12,8 +12,9 @@
     <!--[if IE 8]><link href="<%= asset_path "govuk-template-ie8.css" %>" media="screen" rel="stylesheet" /><![endif]-->
     <link href="<%= asset_path "govuk-template-print.css" %>" media="print" rel="stylesheet" />
 
-    <!--[if IE 8]><link href="<%= asset_path "fonts-ie8.css" %>" media="all" rel="stylesheet" /><![endif]-->
-    <!--[if gte IE 9]><!--><link href="<%= asset_path "fonts.css" %>" media="all" rel="stylesheet" /><!--<![endif]-->
+    <% if content_for?(:fonts_css) %>
```

Comment:
As suggested on Slack a while back:

Would it be simpler for this to be `:exclude_new_transport` (or some other name) and use the existing `:head` variable to include any alternative font.

Reply:
Ah yeah, thanks I knew you'd suggested something but I've been away for two weeks and slack history doesn't go back anywhere near that long 🙄